backend/src/test/java/com/bakdata/conquery/integration/sql/dialect/PostgreSqlIntegrationTests.java

```diff
@@ -87,11 +92,13 @@ public void shouldThrowException() {
 				  .hasMessageContaining("$org.postgresql.util.PSQLException");
 	}
 
-
 	@TestFactory
 	@Tag(TestTags.INTEGRATION_SQL_BACKEND)
-	public List<DynamicNode> sqlBackendTests() {
-		return super.sqlTests(testSqlDialect, sqlConfig);
+	public Stream<DynamicNode> sqlBackendTests() {
```

Comment:
müsste das eigentlich nicht invertiert werden? SQL Tests erzeugen einen Hana/PSQL Case?

Reply:
Ich verstehe was du meinst, aber das würde ich dann ggf. in einem anderen PR machen. Vielleicht sehe ich das auch zu kompliziert, aber die jeweiligen dialekt-spezifischen Testklassen haben z.B. alle eigene `@BeforeAll`/`@AfterAll` hooks und nutzen `@Testcontainer` - all das müsste ich ja dann irgendwie anders lösen, wenn die jeweiligen Klassen nicht selber JUnit-Tests sind. Vielleicht können wir das morgen nochmal kurz thematisieren.